### ISSUE TITLE
Have validators reflect that values are optional by default.

### ISSUE THREAD

**Issue by mthadley** [metal/metal.js#140]
Given the following configuration:

``` js
MyComponent.PROPS = {
  foo: Config.number().isRequired(),
  bar: Config.number()
};
```

It seems that `bar` should be considered optional, since we omitted the `isRequired` call. However, validator warnings will still be thrown if a value is not passed.

It seems like we may want something like `Config.number()` to be implicitly `Config.maybe(Config.number())`.

Any thoughts on this would be greatly appreciated, thanks!

**Comment by mairatma** [comment]
That makes sense, if a value is not listed as required than it should behave as optional, otherwise it'll be very confusing. Your suggestion of having validators work as `maybe` by default is a good one. In this case we may actually not need to expose `Config.maybe` at all then, since that's always implied.

**Comment by mairatma** [comment]
This is now available in version 2.1.3.
